command_linux.go

```diff
@@ -32,10 +33,24 @@ func (r *Runc) command(context context.Context, args ...string) *exec.Cmd {
 	cmd.SysProcAttr = &syscall.SysProcAttr{
 		Setpgid: r.Setpgid,
 	}
-	cmd.Env = os.Environ()
+	cmd.Env = filterEnv(os.Environ(), "NOTIFY_SOCKET") // NOTIFY_SOCKET introduces a special behavior in runc but should only be set if invoked from systemd
```

Comment:
Non-systemd can set this

Reply:
I think `NOTIFY_SOCKET` env is from systemd and I meet the error before. 😂

Reply:
ignore me...

Reply:
I think the `NOTIFY_SOCKET` is used for daemon running in the background. go-runc is just like a tool and I think it is ok to remove it.

Reply:
I'm fine with runc having a special mode for mounting extra things into container but it needs to be more explicit. Then if this feature is found to be useful for `go-runc`, it can have a special option for it.